tests/plugins/templates/manual/insertTemplate.md

```diff
@@ -0,0 +1,27 @@
+@bender-tags: 4.16.2, feature
+@bender-ui: collapsed
+@bender-ckeditor-plugins: wysiwygarea, toolbar, dialog, templates, image, table, format, basicstyles
+
+Insert template and replace all contents
+========================================
```

Comment:
We prefer marking headings with hash and the description should start from the level 2 heading:

```suggestion
## Insert template and replace all contents
```

Please update the headings.